onchain/rollups/test/foundry/inputs/InputBox.t.sol

```diff
@@ -158,13 +158,33 @@ contract InputBoxTest is Test {
         assertEq(inputBox.getNumberOfInputs(_dapp), 0);
     }
 
+    function getMaxInputPayloadLength() internal pure returns (uint256) {
+        // an input blob instance
```

Comment:
I feel like these comments only add noise, since the code is pretty straightforward and easy to understand.

Reply:
I don't see this add noise actually